src/support/polygon-utils.js

```diff
@@ -94,6 +95,23 @@ const PolygonUtils = {
     const color = htmlColors.hex(names[index + 6])
     return color
   },
+  /**
+   * Generate a color (from red to blue) on a linear scale
+   * @param {Number} value The number to be placed on the scale
+   * @param {Number} scaleMax The maximum value of the scale
+   * @returns {String} color
+   */
+  buildPolygonColorFromScale(value, scaleMax) {
+    let modifier = parseInt(255 * (1 -(value/scaleMax)))
+
+    let red = (255 - modifier) * 2
+    let blue = modifier * 2
```

Comment:
```suggestion
    let modifier = parseInt(255 * (value/scaleMax))

    let red = modifier * 2
    let blue = (255 - modifier) * 2
```
although if you leave the red/blue assignments, it is more `blue` which i like better.

But the out- and inmost colors are still very similar.
Probably we should try around with the hsl colors like in the [old client](https://github.com/GIScience/openrouteservice-app/blob/cbbc51bb42541abdc8537f85c8e5b3590f83e5df/app/components/ors-map/ors-map.js#L1492-L1527) where it might be easier to generate more distinct colors?